frontend/src/components/Dropdown/ToolbarDropdown.tsx

```diff
@@ -90,7 +91,11 @@ export const ToolbarDropdown: React.FC<ToolbarDropdownProps> = (props: ToolbarDr
 
   return (
     <>
-      {props.nameDropdown && <span className={dropdownTitle}>{props.nameDropdown}</span>}
+      {props.nameDropdown && (
+        <span id={`${props.id}-name`} className={`${dropdownTitle} ${props.nameDropdownClassName}`}>
```

Comment:
When you have more than a className it is better to use the utility method `classes` from 'typestyle'. This will avoid `undefined` classNames.

Reply:
gracias, I didn't know about this.  Will change.